docs/specification/draft/basic/authorization.mdx

```diff
@@ -91,6 +91,33 @@ as described in [RFC9728 Section 5.1 "WWW-Authenticate Response"](https://datatr
 
 MCP clients **MUST** be able to parse `WWW-Authenticate` headers and respond appropriately to `HTTP 401 Unauthorized` responses from the MCP server.
 
+#### Default Scopes
+
+In addition to the standard Protected Resource Metadata fields defined in [RFC9728](https://datatracker.ietf.org/doc/html/rfc9728),
+MCP servers **MAY** include the optional `scopes_default` extension.
```

Comment:
I believe according to [RFC 6750](https://datatracker.ietf.org/doc/html/rfc6750#section-3) it is allowed to return a `scope` field in a `WWW-Authenticate` header for `401` errors as well as `403` errors. 

If the server returned a `scope` parameter in the initial `WWW-Authenticate` at the same time it returned the `resource_metadata` parameter, then the spec wouldn't need to define the `scopes_default` extension and scope handling for both the initial scope selection and the upgrade flow could be united.

Reply:
That's a really good point, @max-stytch - and it would avoid extra hassle with extending the PRM. Updated the doc.